README.md

```diff
@@ -452,6 +452,22 @@ set HTTP_PROXY=http://127.0.0.1:8888
 
 On Linux, please use `export` other than `set` to set the variables.
 
+# Azure Storage JavaScript Client Library for Browsers
+
+* JavaScript Client Library is currently in Preview. Questions and feedback are welcome in our Slack channel. [![Slack](https://azurestorageslack.azurewebsites.net/badge.svg)]( https://azurestorageslack.azurewebsites.net)
+
+Azure Storage Node.js Client Library is compatible with Browserify. This means you can bundle your Node.js application which depends on the Node.js Client Library using Browserify.
+
+You can also choose to download the JavaScript Client Library provided by us, or generate the library by yourself. Please refer to the [README.md](browser/README.md) under `browser` folder for detailed usage guidelines.
+
+## Downloading Azure Storage JavaScript Client Library
+
+It's recommended to use the Azure Storage JavaScript Client Library provided by us. Please [download the latest library](https://aka.ms/downloadazurestoragejs) from Azure CDN. 
+
+## Generating Azure Storage JavaScript Client Library
+
+We also provide browserify bundle scripts which generate Azure Storage JavaScript Client Library. The bundle script reduces the size of the Storage Client Library by splitting into smaller files, one per storage service. The [README.md](browser/README.md) under `browser` folder describes detailed generating guidelines.
```

Comment:
The [README.md](browser/README.md) under `browser` folder describes detailed generating guidelines. 
==>
For more detailed information, refer to [README.md](browser/README.md) under `browser` folder.